osu.Game.Rulesets.Osu/Objects/OsuHitObject.cs

```diff
@@ -12,7 +12,7 @@ namespace osu.Game.Rulesets.Osu.Objects
 {
     public abstract class OsuHitObject : HitObject, IHasComboInformation, IHasPosition
     {
-        public const double OBJECT_RADIUS = 64;
```

Comment:
not 100% sure about changing this to `float`, but it looks to only be used in FP precision usages at the moment.

Reply:
Shouldn't have been double, +1 from me.